src/pages/AddPost.tsx

```diff
@@ -15,8 +15,12 @@ import {jsx, SxStyleProp} from 'theme-ui';
  */
 export const AddPost: React.FC = () => {
 
+  const wrapperStyle: SxStyleProp = {
+    variant: 'bodyWrapper',
```

Comment:
Mind really quickly informing me what `variant` is for?

Reply:
it's a `theme-ui` thing. Basically, in `theme ui` you can do this thing where you define something that's custom (so like not `colors, fontWeight, lineHeights, etc`) and then you can get all those styles as variant -> https://theme-ui.com/theme-spec#variants.
```ts
export const theme = {
  cool: {
    px: 10,
  }
};

const style = {
  variant: 'cool',
};
```

Reply:
Gotchu, thanks!